src/Components/SubscriptionEditor/SubscriptionEditor.jsx

```diff
@@ -79,14 +80,27 @@ export default class SubscriptionEditor extends React.Component<Props> {
                                 value={subscription.tags}
                                 onChange={nextTags => {
                                     onChange({
-                                        tags: nextTags,
+                                        tags: subscription.any_tags ? null : nextTags,
```

Comment:
Why you need this, if you disabled field?

Reply:
Corrected